packages/zkdb/src/types/transaction.ts

```diff
@@ -11,6 +11,6 @@ export type TDbTransaction = {
   databaseName: string;
```

Comment:
export type TTransaction

Reply:
Should move all type definition to `@zkdb/common` don't define type here.

Reply:
done